### ISSUE TITLE
Please update the device tree with the latest version (the one you use to using to build the new TWRP versions).

### ISSUE THREAD

**Issue by Yilliee** [TeamWin/android_device_samsung_klte#6]
Also please create twrp 3.4 for klte again with the new twrp source as the errors occuring before have been fixed in the latest Twrp Source code.

**Comment by bigbiff** [comment]
Where are the updates, I am not seeing them in this repository.

**Comment by Yilliee** [comment]
> Where are the updates, I am not seeing them in this repository.

They are on twrp.me but this repository is showing as the device tree of klte for TWRP recoveries on TWRP.me.

**Comment by bigbiff** [comment]
I am not sure what you mean.

**Comment by Yilliee** [comment]
> I am not sure what you mean.

Let me rephrase what I wrote before:

Basically I have 2 requests for the maintainer of TWRP for klte:
1) Please make sure that all the local commits of device_samsung_klte have been pushed.
2) Please rebuild TWRP 3.4 for klte as the issues because of which it was taken down have been fixed in the Latest TWRP source.

**Comment by bigbiff** [comment]
The original maintainer hasn't been active in some time. If you have a more specific issue you can open a new issue for this device detailing what that issue is, and maybe someone can fix it some day.
